shared/chat/conversation/info-panel/index.js

```diff
@@ -44,183 +39,222 @@ type InfoPanelProps = {
   // Used by Participant.
   onShowProfile: (username: string) => void,
 
-  // Used by ConversationFooterRow.
+  // Used for conversations.
   onShowBlockConversationDialog: () => void,
   onShowNewTeamDialog: () => void,
 
-  // Used by {Small,Big}HeaderRow.
+  // Used for small and big teams.
   onViewTeam: () => void,
 
-  // Used by BigHeaderRow.
+  // Used for big teams.
   onLeaveConversation: () => void,
   onJoinChannel: () => void,
 } & HeaderHocProps
 
-type ConversationFooterRow = {
-  type: 'conversation footer',
-  key: 'CONVERSATION FOOTER',
+type ParticipantRow = ParticipantInfo & {
+  type: 'participant',
```

Comment:
these `type`s are repeated a bunch, I think an enum somewhere would make these easier to maintain

Reply:
I think I had an enum (that used `$PropertyType`) but ended up never needing it. Flow is smart enough to detect if those literal strings gets typoed, anyway.

Reply:
Per our chat, I tried defining `const FooType = 'foo'` constants and using them, but somehow this made flow *more* confused than with just literal strings. So I'll leave it as is.